runtime/gc_base/GCExtensions.cpp

```diff
@@ -333,7 +333,11 @@ MM_GCExtensions::needScanStacksForContinuationObject(J9VMThread *vmThread, j9obj
 	 */
 	if (started && !finished) {
 		Assert_MM_true(NULL != continuation);
-		needScan = !VM_VMHelpers::isContinuationMountedOrConcurrentlyScanned(continuation, isGlobalGC);
+		if (beingMounted) {
```

Comment:
Add a comment:

.... for sliding compacts to avoid double slot fixups

If continuation is currently being mounted by this thread, we must be in pre/postMount callback and must scan.

For fully STW GCs,....